data.sql

```diff
@@ -5,7 +5,16 @@ INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
-    VALUES ('Agumon', '2020-02-03', 0, true, 10.23 ),
-        ('Gabumon', '2018-11-15', 2, true, 8 ),
-        ('Pikachu', '2021-01-07', 1, false, 15.04 ),
-        ('Devimon', '2017-05-12', 5, true, 11 );
+VALUES ('Agumon', '2020-02-03', 0, true, 10.23 ),
+('Gabumon', '2018-11-15', 2, true, 8 ),
+('Pikachu', '2021-01-07', 1, false, 15.04 ),
+('Devimon', '2017-05-12', 5, true, 11 );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, false, 11 ),
+('Plantmon', '2021-11-15', 2, true, 5.7),
+('Squirtle', '1993-04-02', 3, false, 12.13),
```

Comment:
- [ ] The weight here should be `-12.13` and not `12.13` as seen in the example.